classes/opponentSpawner.lua

```diff
@@ -29,6 +29,28 @@ function opponentSpawner.new(startIndex)
 		classes.opponentPhaseBullethell4,
 	}
 
+	self.introText =
+	{
+		"Hello World", -- Tutorial
+		"", -- Space Invaders
+		"", -- Bouncing
+		"", -- Orbit
+		"", -- DVD
+		"", -- Charge
+		"", -- Charge Shot
+		"", -- Chase
+		"", -- Teleport
+		"This is the part where he kills you", -- Portals
+		"", -- Firework
+		"", -- Weakspot
+		"", -- Ducks
+		"", -- Fakeout
+		"", -- Bullethell 1
+		"", -- Bullethell 2
+		"", -- Bullethell 3
+		"", -- Bullethell 4
+	}
```

Comment:
We can use this approach. But I'd prefer instead that we add a line to the top of each phase's file with an optional string

Reply:
For example, adding a line 10 to `opponentPhaseNormal.lua`:

```lua
opponentPhaseNormal.NUM_LIVES = 4
opponentPhaseNormal.SPAWN_Y = 100
opponentPhaseNormal.DEFAULT_BULLET_COOLDOWN = 1
-- vvv  THE NEW LINE  vvv
opponentPhaseNormal.INTRO_TEXT = "Hello World"
```